classes/Templates.php

```
@@ -101,4 +101,24 @@ public static function copyTemplate(int $campaignId, bool $testMode): string|boo
+        );
+
+        foreach ($theme_template_paths as $template_path) {
+            if (file_exists($template_path)) {
+                return $template_path;
+            }
+        }
+
+        return MAWIBLAH_TEMPLATES_PATH. '/missingTemplate.php';
+    }
+
+    public static function loadTemplate(string $templatePath, mixed $data){
```

PHPMD (2.15.0)

Avoid unused parameters such as '$data'. (undefined)

(UnusedFormalParameter)